definitions/ext-maxdb/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: EXT
+type: MAXDB
```

Comment:
should we prefix this with sap to keep consistent or there is a reason to not do that?

Reply:
Maxdb is a database. During the discussion with the entity-platform team, it is indicated to us that it is better to keep the database entity type generic, such as reusing the existing  ext-db2 type if we can. So we are proposing the new DB entity types such as EXT_MAXDB, EXT-HANADB, EXT-ORACLE, and EXT-SQLSERVER.  Ext-MAXDB is the first one being generated, the rest will come later when we have customers using those databases.

Reply:
sounds great!